website/config.toml

```diff
@@ -77,6 +77,13 @@ languageName = "日本語(Japanese)"
 contentDir = "content/ja"
 weight = 4
 
+[languages.es]
+title = "CNCF TAG App Delivery"
+description = "El TAG App Delivery se centra en habilitar proyectos e iniciativas relacionadas con la entrega de aplicaciones nativas en la nube, incluyendo su construcción, manejo y operativa."
```

Comment:
"El TAG App Delivery se centra en habilitar proyectos e iniciativas **relacionados** con el despliegue de aplicaciones cloud-native, incluyendo su construcción, manejo y operativa." 

A couple of comments @guillermotti :
- it's better if we use cloud-native due to in Spanish is no need to translate this as "aplicaciones nativas en la nube"
- instead of "entrega" the word of "despliegue" is more accurated

Hope that helps!